spec/happy-flow/backend.md

```diff
@@ -114,16 +223,16 @@ Check if the payment due date is reached, trigger the benefit payment to the cit
   ]
 }
 ```
-1. Globally unique ID which is coming from [driver app](happy-flow.md#implementation).
-2. ID which is coming from [driver app](happy-flow.md#implementation) to identify the origination of the request.
+1. Globally unique ID which is coming from [driver app](main.md#implementation).
+2. ID which is coming from [Information Mediator](https://github.com/GovStackWorkingGroup/sandbox-bb-information-mediator) to identify the origination of the request.
```

Comment:
might not be IM, every request to api endpoints will be routed through IM, and in that case this id will be the same ...  So more logical case is that this ID will be defined by the Calling BB in our case DriverApp. Same in other places related with Payment APIs